src/weather_model_graphs/visualise/plot_2d.py

```diff
@@ -13,7 +13,7 @@ def nx_draw_with_pos(g, with_labels=False, **kwargs):
     if ax is None:
         _, ax = plt.subplots(figsize=(10, 10))
     networkx.draw_networkx(
-        ax=ax, G=g, pos=pos, hide_ticks=False, with_labels=with_labels, **kwargs
+        ax=ax, G=g, pos=pos, with_labels=with_labels, **kwargs
```

Comment:
`hide_ticks` seems to be new from networkx version 3.3 (https://networkx.org/documentation/stable/reference/generated/networkx.drawing.nx_pylab.draw_networkx.html). Looks like this is the version we require https://github.com/mllam/weather-model-graphs/blob/2456064543200af1adfd17db0d1ccbb5f7441ca8/pyproject.toml#L11
so if you upgrade to 3.3 then this change should not be necessary.

Reply:
Oh, I should have checked that. I will revert the change.